backend/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ repositories {
 tasks {
     withType(KotlinCompile::class.java) {
         kotlinOptions {
-            freeCompilerArgs = listOf("-Xjsr305=strict", "-Xenable-jvm-default")
+            freeCompilerArgs = listOf("-Xjsr305=strict", "-Xjvm-default=enable")
```

Comment:
Out of curiosity,c an you explain why this changed?

Reply:
See https://blog.jetbrains.com/kotlin/2018/06/kotlin-1-2-50-is-out/ (and look for "@JvmDefault binary compatibility mode")